src/components/InteractiveBlock/InteractiveBlock.tsx

```diff
@@ -2,11 +2,60 @@ import React from 'react';
 import { InteractiveBlockProps } from '../../interfaces/interfaces';
 import './InteractiveBlock.css';
 
-export const InteractiveBlock: React.FC<InteractiveBlockProps> = ({ isPlaying, id }) => {
+export const InteractiveBlock: React.FC<InteractiveBlockProps> = ({
```

Comment:
Could we choose a bit better name for the component? Interactiveblock for me sounds too abstract